department-of-veterans-affairs/va.gov-cms
# VACMS-4075 Unique validation for content types

## Description
See #4075

## Testing done
Behat

## QA steps
It's probably overkill, but we could create two of each of the content types to which we've added the `field_enforce_unique_combo` field, and duplicate the values in the unique value combo fields, then verify that a notice is created when trying to save the second one. 
We could probably just spot check a few: E.g.: 
- Create two Facility Health Services with the same field_service_name_and_descripti value, and visually verify that a notice is thrown when saving the second. 
- Create two Story Listing nodes with the same field_office value, and visually verify that a notice is thrown when saving the second. 
- Create two Operating Status nodes with the same field_office value, and visually verify that a notice is thrown when saving the second. 
- Create two Locations Listing nodes with the same field_office value, and visually verify that a notice is thrown when saving the second. 

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
